keps/sig-node/3675-resource-plugin-manager/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Pluggable Resource Management
+kep-number: 3675
+authors:
+  - "@obiTrinobiIntel"
+  - "@catblade"
+  - "@adrianhoban"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: implementable
+creation-date: "2023-02-06"
+reviewers:
+  - "@swatisehgal"
+  - "@klueska"
+approvers:
+  - "@sig-node-leads"
+
+see-also: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.27"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.27"
+  beta: "v1.30"
+  stable: "v1.33"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: CCIManager
```

Comment:
In the proposal doc (README), the feature gate was specified as `CCIResourceManager`. Please ensure that the feature gate name is consistent.